# This yaml specifies the origins and how to process each point cloud in the datasets directory
# The following fields should be specified:

# This enum is used for specifying the obstacle that should be dynamically
# made with the RealSense camera. All supported obstacles should be
# specified here
# uint8 STAIRS_UP=0
# uint8 STAIRS_DOWN=1
# uint8 RAMP_UP=2
# uint8 RAMP_DOWN=3
# uint8 selected_gait
#
# The link to transform to, is generally set to either "foot_left" or "foot_right"
# string frame_id_to_transform_to
#
# Whether the pointcloud has been taken from the front camera position or the back camera position
# bool from_back_camera
bankje-centrum-01-08-2021-1.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
bankje-centrum-01-08-2021-2.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
bankje-centrum-01-08-2021-3.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
bankje-centrum-01-08-2021-4.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
bankje-centrum-met-merel-01-08-2021-1.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
route-bankje-centrum-03-08-2021-1.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
route-bankje-centrum-03-08-2021-2.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
route-bankje-centrum-03-08-2021-3.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
route-bankje-centrum-03-08-2021-4.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
chair1.ply:
  realsense_category: 4
  subgait_name: "right_open"
  from_back_camera: true
  from_realsense_viewer: false
chair_high.ply:
  realsense_category: 4
  subgait_name: "right_open"
  from_back_camera: true
  from_realsense_viewer: false
chair_low.ply:
  realsense_category: 4
  subgait_name: "right_open"
  from_back_camera: true
  from_realsense_viewer: false
chair_medium.ply:
  realsense_category: 4
  subgait_name: "right_open"
  from_back_camera: true
  from_realsense_viewer: false
chair_wooden_low.ply:
  realsense_category: 4
  subgait_name: "right_open"
  from_back_camera: true
  from_realsense_viewer: false
chair_wooden_medium.ply:
  realsense_category: 4
  subgait_name: "right_open"
  from_back_camera: true
  from_realsense_viewer: false
chair_wooden_high.ply:
  realsense_category: 4
  subgait_name: "right_open"
  from_back_camera: true
  from_realsense_viewer: false
door.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
from-ramp.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
ramp2.ply:
  realsense_category: 2
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
ramp-from-stand-with-crutch.ply:
  realsense_category: 2
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
ramp-from-stand-with-thijs.ply:
  realsense_category: 2
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
ramp-from-stand-without-thijs.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
ramp-leaning.ply:
  realsense_category: 2
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
ramp_mini.ply:
  realsense_category: 2
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: false
ramp_mini2.ply:
  realsense_category: 2
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: false
single_step1.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: false
single_step_chair_background.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: false
single_step_high_undeep.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: false
stairs1.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
stairs2.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
stairs3.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
stairs4.ply:
  realsense_category: 0
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
stairs_down1.ply:
  realsense_category: 1
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
stairs_down2.ply:
  realsense_category: 1
  subgait_name: "right_open"
  from_back_camera: false
  from_realsense_viewer: true
zitblok-centrum-01-08-2021-1.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
zitblok-centrum-voor-uitvoeren-01-08-2021-1.ply:
  realsense_category: 4
  subgait_name: "prepare_sit_down"
  from_back_camera: true
  from_realsense_viewer: false
increasing_ramp.ply:
  realsense_category: 2 
  frame_id_to_transform_to: "foot_right"
  from_back_camera: false
  from_realsense_viewer: true
